Sekoia.io/sekoiaio/triggers/alerts.py

```diff
@@ -12,33 +12,37 @@
 
 class SecurityAlertsTrigger(_SEKOIANotificationBaseTrigger):
     # List of alert types we can handle.
-    HANDLED_EVENT_SUB_TYPES = [
-        "alert-created",
-        "alert-updated",
-        "alert-status-changed",
-        "alert-comment-created",
-    ]
-
-    def handle_alert(self, message):
+    HANDLED_EVENT_SUB_TYPES = [("alert", "created"), ("alert", "updated"), ("alert-comment", "created")]
+
+    def handle_event(self, message):
         """Handle alert messages.
 
-        Only a few event are considered (`alert-created`,
-        `alert-updated`, `alert-status-changed`,
-        `alert-comment-created`). If a valid evnet is handled, then
-        enrich event from `sicalertapi` to retrieve its status, its
-        short id, etc. Finally, send message to the Symphony workflow.
+        Only a few event are considered (`alert:created`,
+        `alert:updated`, `alert-comment:created`). If a valid evnet is
+        handled, then enrich event from `sicalertapi` to retrieve its
+        status, its short id, etc. Finally, send message to the
+        Symphony workflow.
 
         """
         alert_attrs = message.get("attributes", {})
-        alert_event: str | None = alert_attrs.get("event")
-        alert_uuid: str | None = alert_attrs.get("alert_uuid")
+        event_type: str | None = message.get("type")
+        event_action: str | None = message.get("action")
+
+        # Ignore alert “sub event” types that we can’t (yet) handle.
+        if (event_type, event_action) not in self.HANDLED_EVENT_SUB_TYPES:
+            return
+
+        alert_uuid: str | None = None
+        if event_type == "alert":
+            alert_uuid = alert_attrs.get("uuid")
+        elif event_type == "alert-comment":
+            alert_uuid = alert_attrs.get("alert_uuid")
 
         # Is the notification in a format we can understand?
-        if not (alert_event and alert_uuid):
+        if not (event_type and event_action and alert_uuid):
```

Comment:
We already checked that `event_type` and `event_action` are in `HANDLED_EVENT_SUB_TYPES` so I guess they can't be None

```suggestion
        if not alert_uuid:
```

Reply:
Thanks, PR was updated.